README.md

```diff
@@ -23,7 +23,7 @@ pip install -r requirements.txt
 
 ## Documentation
 
-The documentation for RIBS can be found at [tbd]
+The documentation for RIBS can be found at https://lithekod.github.io/snake-ribs/
```

Comment:
Shouldn't this be a link?

Reply:
Yes, I assumed links were detected automagically when I wrote that, i'll take care of it

Reply:
Oh wait, this is markdown so it should be automatically made a link I think